rasa/server.py

```diff
@@ -1152,7 +1157,9 @@ async def evaluate_intents(
                     test_data, config_file, int(cross_validation_folds)
                 )
         else:
-            test_data = _test_data_file_from_payload(request, temporary_directory)
+            payload = _nlu_training_payload_from_json(request, temporary_directory)
```

Comment:
Should we add the deprecation warning here too?

Reply:
Fortunately it appears automatically because the method calls other methods that contain it